src/encoded/tests/test_audit_file.py

```diff
@@ -912,3 +912,123 @@ def test_audit_self_matching_md5sum(testapp, file7):
         errors_list.extend(errors[error_type])
     assert any(error['category'] == 'inconsistent matching_md5sum'
                for error in errors_list)
+
+
+def test_audit_incorrect_index(testapp, fastq_index, single_fastq_indexed, base_experiment):
+    res = testapp.get(fastq_index['@id'] + '@@index-data')
+    errors = res.json['audit']
+    errors_list = []
+    for error_type in errors:
+        errors_list.extend(errors[error_type])
+    assert 'inconsistent index file' not in (error['category']
```

Comment:
this is the test of a correct audit, right?   Should be a comment or something because the test is still named "incorrect_index"

Reply:
I reorganized and wrote more tests to make this clear- test_audit_correct_index gives 3 acceptable states (1 SE fastq, 2 PE fastq paired with each other, 1 PacBio fastq); test_audit_incorrect_index gives all 7 incorrect error states. Some of the error states could have been combined into fewer tests, but the each error is absolutely explicit as written (in the new version)

Reply:
Also it was definitely worth going through more explicitly, the way I was finding 2 PacBio files wasn't quite correct before! But that should be fixed now, it's the only new change to the actual audit code